src/components/SightingPage.js

```diff
@@ -0,0 +1,113 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import { useParams, useNavigate, useOutletContext } from "react-router-dom";
+import { BACKEND_URL } from "../constants";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import Typography from "@mui/material/Typography";
+import { Button, TextField } from "@mui/material";
+import "../App.css";
+
+export default function SightingPage() {
+  const params = useParams();
+  const id = params.sightingId;
+  const [sighting, setSighting] = useState();
+  const [comments, setComments] = useState();
+  const [content, setContent] = useState(null);
+  const [categories, setCategories] = useState([]);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    const takeSightings = async () => {
+      try {
+        const sightingData = await axios.get(`${BACKEND_URL}/sightings/${id}`);
+        console.log(id);
+        const sightingInfo = sightingData.data;
+
+        setSighting(sightingInfo);
+
+        setCategories(sightingInfo.categories);
+
+        const commentData = await axios.get(
+          `${BACKEND_URL}/sightings/${id}/comments`
```

Comment:
what if the id is undefined? you will not get a sighting above, and also no comments. Best to validate that first